apps/console/src/extensions/configs/index.ts

```diff
@@ -21,5 +21,6 @@ export * from "./application";
 export * from "./identity-provider";
 export * from "./common";
 export * from "./server-configuration";
-export * from "./userstores";
-export * from "./scim";
+export * from "./userstores"
+//Do not import scim here cause cyclic dependency
+// export * from "./scim";
```

Comment:
shall we remove commented code?

Reply:
+1

Reply:
@ravindudsilva Lets add the comment to the usages. i.e `claim-management-constants.ts ` etc.

Reply:
Fixed in the commit c94a4280ff072f79f21d8f47f213609e26035952